Makefile

```
@@ -328,18 +337,28 @@ test-experimental-e2e: GO_BUILD_EXTRA_FLAGS := -cover
+
 .PHONY: prometheus
 prometheus: PROMETHEUS_NAMESPACE := olmv1-system
-prometheus: PROMETHEUS_VERSION := v0.83.0
-prometheus: $(KUSTOMIZE) #EXHELP Deploy Prometheus into specified namespace
+prometheus: PROMETHEUS_CHART_VERSION := 86.2.2
+prometheus: $(HELM) #EXHELP Deploy Prometheus into specified namespace
 ifeq ($(strip $(E2E_SUMMARY_OUTPUT)),)
 	@echo "E2E_SUMMARY_OUTPUT unset; skipping prometheus deployment"
 else
-	./hack/test/install-prometheus.sh $(PROMETHEUS_NAMESPACE) $(PROMETHEUS_VERSION) $(VERSION) $(PROMETHEUS_VALUES)
+	$(HELM) upgrade --install prometheus oci://ghcr.io/prometheus-community/charts/kube-prometheus-stack \
```

checkmake (0.3.2)

[warning] Target body for "$(HELM) upgrade --install prometheus oci" exceeds allowed length of 5 lines (8).

(maxbodylength)